src/aeonlib/ocs/target_models.py

```diff
@@ -13,44 +13,34 @@ class SiderealTarget(BaseModel):
     model_config = ConfigDict(validate_assignment=True)
     name: Annotated[str, StringConstraints(max_length=50)] = "string"
     """The name of this Target"""
-    type: Literal["ICRS", "HOUR_ANGLE"]
+    type: Literal["ICRS", "HOUR_ANGLE", "ALTAZ"]
```

Comment:
So, this is a little complicated. The required fields for:

- ICRS are ra/dec
- HOUR_ANGLE are hour_angle/dec
- ALTAZ are altitude/Azimuth

All of these are perfectly reasonable ways to point at a target, but might have different implications.
For instance, pointing in ALT/AZ usually implies that you aren't tracking on the sky.

Reply:
I see, these could be made sub-classes with only the relevant fields if we want the API to be as concise as possible.